django/views/generic/dates.py

```diff
@@ -300,7 +300,11 @@ def _make_single_date_lookup(self, date):
 
 
 class BaseDateListView(MultipleObjectMixin, DateMixin, View):
-    """Abstract base class for date-based views displaying a list of objects."""
+    """
+    A base view for date-based views displaying a list of objects.
```

Comment:
The previous line was probably clearer and more concise:
```suggestion
    Abstract base class for date-based views displaying a list of objects.
```
or maybe

```suggestion
    Base class for date-based views displaying a list of objects.
```

Reply:
It was but as earlier @carltongibson quoted that all the base views should have a consistency in the wordings, i considered that and made the changes. I will make the minor sentence changes you recommended as those are important. Rest what do you think , Should we turn it back? eitherways it gives a more concrete message.